README.md

```diff
@@ -1,223 +1,104 @@
+- **Enhanced Telemetry**: To help us improve RudderStack, we collect performance and diagnostic metrics about how you use RudderStack, and how it is working. **No customer data is present in the metrics**. For technical details, please check out our wiki page on [Telemetry](https://github.com/rudderlabs/rudder-server/wiki/Telemetry).
 
-## Start RudderStack with the workspace config file
 
-- Download the workspace config file on your machine.
-- In `docker-compose.yml`, uncomment `volumes` section under `backend` service. Specify the path to your workspace config.
-- In `build/docker.env`, set the environment variable `RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE=true`
+# RudderStack Architecture
 
----
+RudderStack has two major components, namely:
+- The Control Plane: Allows you to manage the configuration of your sources and destinations
+- The Data Plane:  This is the core engine that is responsible for:
 
-# RudderStack Architecture
+   - Receiving and buffering the event data
+   - Transforming the event data into the required destination format, and
+   - Relaying it to the destination
 
-The following is a brief overview of the major components of RudderStack.
-![image](https://user-images.githubusercontent.com/52487451/64673994-471beb00-d48d-11e9-854f-2c3fbc021e63.jpg)
+For a detailed understanding of the RudderStack architecture, please check our [documentation](https://docs.rudderstack.com/getting-started/rudderstack-data-plane-architecture)
 
-## RudderStack Control Plane
+# RudderStack Setup
 
-The UI to configure the sources, destinations etc. It consists of
+The easiest way to set up and use RudderStack is through our hosted service. For detailed instructions, please refer to our documentation on using the [RudderStack Hosted Service](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-a-hosted-demo-account).
 
-**Config backend**: This is the backend service that handles the sources, destinations and their connections. User management and access based roles are defined here.
+You can also set up and use RudderStack on your platform of choice. Please refer to the following guides for detailed instructions:
 
-**Customer webapp**: This is the front end application that enables the teams to set up their customer data routing with RudderStack. These will show you high-level data on event deliveries and more stats. It also provides access to custom enterprise features.
+- [Setting up RudderStack on Docker](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-docker)
+- [Setting up RudderStack on Kubernetes](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-kubernetes)
+- [Setting up RudderStack on a Native Installation](https://docs.rudderstack.com/getting-started/installing-and-setting-up-rudderstack#setup-instructions-for-a-native-installation)
```

Comment:
These 3 links are redirecting to the same page (https://docs.rudderstack.com/administrators-guide/installing-and-setting-up-rudderstack). That page contains the link to the relevant docs. Can we directly link to those docs instead?

Reply:
Sure, I'll fix this.